GitCommands/Settings/AppSettings.cs

```diff
@@ -1842,26 +1773,7 @@ public static Font GetFont(string name, Font defaultValue)
             return SettingsContainer.GetFont(name, defaultValue);
         }
 
-        public static void SetColor(string name, Color? value)
-        {
-            SettingsContainer.SetColor(name, value);
-        }
-
-        public static void SetColor(AppColor name, Color? value)
-        {
-            SettingsContainer.SetColor(name.ToString().ToLowerInvariant() + "color", value);
-        }
-
-        public static Color? GetColor(string name)
-        {
-            return SettingsContainer.GetColor(name);
-        }
-
-        public static Color GetColor(string name, Color defaultValue)
-        {
-            return SettingsContainer.GetColor(name, defaultValue);
-        }
-
+        [Obsolete("AppSettings is no longer responsible for colors, ThemeModule is")]
         public static Color GetColor(AppColor name)
         {
             return SettingsContainer.GetColor(name.ToString().ToLowerInvariant() + "color", AppColorDefaults.GetBy(name));
```

Comment:
Plans to delete it later?

Reply:
When this is deleted, users who skip update until then will not get their custom colors migrated. So I was thinking to leave this indefinitely.